packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -292,6 +306,9 @@ namespace KB = KokkosBatched::Experimental;
       typedef typename Kokkos::Details::ArithTraits<scalar_type>::val_type impl_scalar_type;
       typedef typename Kokkos::ArithTraits<impl_scalar_type>::mag_type magnitude_type;
 
+      typedef typename BlockTridiagScalarType<impl_scalar_type>::type btdm_scalar_type;
```

Comment:
Note to self: This should work fine with complex arithmetic enabled, since `impl_scalar_type` would be `Kokkos::complex<T>` in that case, not `std::complex<T>`.